lib/statsd.js

```diff
@@ -91,6 +102,10 @@ var Client = function (host, port, prefix, suffix, globalize, cacheDns, mock,
     global.statsd = this;
   }
 
+  if (options.protocol === 'tcp') {
+    this.socket.setKeepAlive(true);
+  }
```

Comment:
Shouldn't this be a part of the `createSocket` "method"?

Reply:
Addressed in https://github.com/brightcove/hot-shots/pull/79/commits/69de8cbfcad688e76ea621dcb5303250170f3c09.